### PR TITLE
[Excalibur] Original projection curve chart

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Adds the Original Projection curve chart to the `ResponseImpactDashboard`. Also includes the original projection static data object.

Original projection input values are from the PRD:

- Facilities staff cases: 1
- Facilities staff population: [current saved number]
- Incarcerated individuals: 
- Cases: 1 
- Population: If type=state prison, static population default (Column J of data model incarceration data)
- If type = county jail, static population default (Column I of incarceration model data)
- Population turnover: 0%
- Facility occupancy rate: 100%
- Bunk-style housing: 15%
- Rate of spread: Initial Rt value if we have the data, otherwise fall back to 3.7
- Planned releases: None

Notes:
-I will add in the initial Rt value for Rate of Spread later when it is available, and switch on `systemType` for population numbers in another issue.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Related to #240 and #237 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-response-impact/ResponseImpactDashboard.tsx [hunk 1]
```diff
@@ -104,19 +111,19 @@ function getCurveInputs(modelInputs: EpidemicModelState[]) {
...
-    return (sumHospitalBeds += input.hospitalBeds || 0);
-  });
-  return sumHospitalBeds;
-}
-
 const ResponseImpactDashboard: React.FC = () => {
   const { data: localeDataSource } = useLocaleDataState();
   const [scenarioState] = useScenario();
   const [curveInputs, setCurveInputs] = useState([] as CurveFunctionInputs[]);
+  const [originalCurveInputs, setOriginalCurveInputs] = useState(
```
Comment (jessex) [inline src/page-response-impact/ResponseImpactDashboard.tsx:118]: If we call these `originalCurveInputs` and `originalWhatever`, let's also rename the variables for the current projection to `currentCurveInputs` and `currentWhatever` to match.

**Comment by jovergaag** [comment]
@jessex This is ready for another look. I know there are some of Daniela's commits in there from before her squash/merge. I tried squashing these to clean up this commit history but it got pretty gnarly since we've been so intertwined. Let me know if you would prefer I clean that up, otherwise, I can save some time and leave it as is.

**Comment by rocktacular** [comment]
> * If type = county jail, static population default (Column I of incarceration model data)

I'm not sure we are importing Column I (`Jail Population (2017)`) anywhere. But it looks like we're only using `totalPrisonPopulation` anyway?

File: src/impact-dashboard/EpidemicModelContext.tsx [hunk 2]
```diff
@@ -152,6 +152,8 @@ export function getLocaleDefaults(
     totalIncarcerated:
       dataSource.get(stateCode)?.get(countyName)?.totalIncarceratedPopulation ||
       0,
+    totalPrisonPopulation:
+      dataSource.get(stateCode)?.get(countyName)?.totalPrisonPopulation || 0,
```
Comment (rocktacular) [inline src/impact-dashboard/EpidemicModelContext.tsx:156]: won't this always be 0, since `Col J` only has values for states but not counties?

Comment (jovergaag) [reply src/impact-dashboard/EpidemicModelContext.tsx:156]: The default value for `countyName` in the params declaration is "Total". So in the dataSource object for the given stateCode, without a countyName given, it pulls from the `['Total']` key which is correct.

Screen Shot 2020-05-01 at 10 18 20 AM

**Comment by jovergaag** [comment]
> > * If type = county jail, static population default (Column I of incarceration model data)
> 
> I'm not sure we are importing Column I (`Jail Population (2017)`) anywhere. But it looks like we're only using `totalPrisonPopulation` anyway?

@rocktacular You are right, we are not importing Column I anywhere yet. I figured we could do that import in the PR that addresses issue of switching the population data based on `systemType`.  Right now, the original projection data is hard coded to 'State Prison' so didn't see a need to import the jail populations yet. Thanks for the sharp eyes on this!!

**Pull request merged**